### ISSUE TITLE
Default learning rate on pretrained layers?

### ISSUE THREAD

**Issue by mehak25** [ThilinaRajapakse/simpletransformers#1570]
I understand we can specify the learning rate for each layer using custom_layer_parameters.
But what is the behavior of simpletransformer training by default? If I only mention learning_rate in model_args, does that apply to all layers in pretrained roberta-large or only the regression layer in it?
